tools/interactive/interactivetool_askomics.xml

```diff
@@ -1,4 +1,5 @@
 <tool id="interactive_tool_askomics" tool_type="interactive" name="AskOmics" version="4.3.1">
+    <icon src="askomics.png" />
```

Comment:
@dannon in Vis we are calling it `logo` I think. The changes are just in dev I think. Should we change that to stay consistent?

Reply:
@bgruening I don't have a strong inclinatino either way -- on the one hand, icon makes more sense than logo to me and would have been a better choice originally, but if we want to be consistent with viz that's fine too.  Whatever @davelopez wants to do here is fine with me.

Reply:
I also don't have a strong opinion on the name, but maybe `icon` does make a bit more sense to me, too. A logo can have an irregular size, contain text, etc., while an icon usually has a fixed size and is easier to integrate in the UI (it can be a simplified version of a logo).

In any case, I agree that consistency is important. We are probably not restricting the size of the icons anyway, so using `logo` can be fine as long as the proportions are fine and it doesn't look too stretched or disrupted.

Reply:
We could also leave it as `icon`, and I could change it for visualizations, but this would require changing the plugins api response from `logo` to `icon`. It would be a minimal change, and I doubt that anyone other than us is using this api, its not a fastapi route yet.